### PR TITLE
bug 1571812 ca.crt with multiple CA certificates

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1571812

@jianlinliu, PTAL

@openshift/team-documentation, PTAL

### CODE REVIEW

File: dev_guide/builds/build_inputs.adoc [hunk 1]
```diff
@@ -569,24 +569,33 @@ $ oc create secret generic <secret_name> \
...
+
+Complete one of the following processes:
+
+* Create a secret with a CA certificate file (recommended).
+.. If your CA uses Intermediate Certificate Authorities, combine the 
+certificates for all CAs in a *_ca.crt_* file. Run the following command:
 +
 ----
-$ oc create secret generic mycert --from-file=ca.crt=</path/to/file> <1>
+"$ cat intermediateCA.crt intermediateCA.crt rootCA.crt > ca.crt"
```
Comment (ahardin-rh) [inline dev_guide/builds/build_inputs.adoc]: Should the " be before the $?

**Comment by ahardin-rh** [comment]
Just one question from me. Otherwise, looks good! :+1:

**Comment by kalexand-rh** [comment]
The change was approved on the source bug, and I've made updates after review. Merging. :)

**Comment by kalexand-rh** [comment]
3.4: https://github.com/openshift/openshift-docs/pull/9693
3.5: https://github.com/openshift/openshift-docs/pull/9692
3.6: https://github.com/openshift/openshift-docs/pull/9691
3.7: https://github.com/openshift/openshift-docs/pull/9690

**Pull request merged**